packages/opentelemetry-plugin-grpc/src/utils.ts

```diff
@@ -17,6 +17,11 @@
 import { CanonicalCode, Status } from '@opentelemetry/api';
 import * as grpcTypes from 'grpc'; // For types only
 
+/**
+ * Metadata key used to denote an outgoing opentelemetry request.
+ */
+const _otRequestHeader = 'x-opentelemetry-outgoing-request';
```

Comment:
nit: This seems to be getting defined in [a lot of separate places](https://github.com/open-telemetry/opentelemetry-js/search?q=x-opentelemetry-outgoing-request&unscoped_q=x-opentelemetry-outgoing-request). @open-telemetry/javascript-approvers Should it be moved to somewhere common like /core or /tracing or /semantic-conventions?

Reply:
Once I get consensus here I can make the change thats required

Reply:
I think just keeping it here is fine _for now_, because a better solution in the form of https://github.com/open-telemetry/opentelemetry-js/pull/1344 is coming.